charts/node-ipam-controller/Chart.yaml

```diff
@@ -17,4 +17,4 @@ name: node-ipam-controller
 description: A Helm chart for node-ipam-controller
 type: application
 version: 0.0.2
-appVersion: 0.0.2
+appVersion: 0.0.1
```

Comment:
is this version downgrade expected?

Reply:
I bumped it initially as I expected changes in the go code, but then downgraded again because only auxiliary scripts and charts were changed.